modules/charon-core/src/main/java/org/wso2/charon3/core/extensions/UserManager.java

```diff
@@ -89,8 +106,24 @@ public Group getGroup(String id, Map<String, Boolean> requiredAttributes)
     public void deleteGroup(String id)
             throws NotFoundException, CharonException, NotImplementedException, BadRequestException;
 
-    default List<Object> listGroupsWithGET(Node node, int startIndex, int count, String sortBy,
-                                          String sortOrder, String domainName, Map<String, Boolean> requiredAttributes)
+    default List<Object> listGroupsWithGET(Node node, Integer startIndex, Integer count, String sortBy,
+            String sortOrder, String domainName, Map<String, Boolean> requiredAttributes)
+            throws CharonException, NotImplementedException, BadRequestException {
```

Comment:
add new line

Reply:
Fixed in https://github.com/wso2/charon/pull/263